catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -246,9 +265,11 @@ private CreateResponse doCreate(CreateRequest createRequest)
       ingestError = iee;
       throw iee;
     } catch (StopProcessingException see) {
+      INGEST_LOGGER.debug("Error during pre ingest", see);
```

Comment:
✏️ Instead of adding this debug statement and the one in the other catch block, I think we should remove the debug statement that's already in the IngestException catch block. These debug logs are providing the same information that the info log (from the finally block) already provides.

Reply:
fixed